tests/pdctl/scheduler/scheduler_test.go

```diff
@@ -530,3 +530,28 @@ func mightExec(re *require.Assertions, cmd *cobra.Command, args []string, v inte
 	}
 	json.Unmarshal(output, v)
 }
+
+func TestMicroservice(t *testing.T) {
```

Comment:
How to know if the output comes from the scheduling server?

Reply:
Because we have already disabled PD scheduling in api mode? How about adding info in the response header for further confirmation?

Reply:
How about using a more concrete name? `TestMicroservice` is too general.